packages/office-addin-lint/src/lint.ts

```diff
@@ -11,7 +11,7 @@ const eslintPath = require.resolve("eslint");
 const prettierPath = require.resolve("prettier");
 const eslintDir = path.parse(eslintPath).dir;
 const eslintFilePath = path.resolve(eslintDir, "../bin/eslint.js");
-const prettierFilePath = path.resolve(prettierPath, "../bin-prettier.js");
+const prettierFilePath = path.resolve(prettierPath, "../bin/prettier.cjs");
```

Comment:
I'm not very familiar with the prettier setup . . . what is effect of this (this is not a file in this package itself)?

Reply:
It's in the prettier code. I suppose you could ask whether the command should use npx instead, but the change here is just to match changes within prettier itself for its code.

Reply:
Makes sense . . . I just haven't seen that file used anywhere so I wasn't sure how it was used.